### PR TITLE
Add SubHeadline Component to Storybook

### PR DESCRIPTION
**Pull request by colefriedlaender**
SubHeadline Component

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
LGTM 💯 
Heroku deployment would be nice, so the components can be seen

File: components/SubHeadline/SubHeadline.module.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.container {
+  height: 70px;
```
Comment (phlgr) [inline components/SubHeadline/SubHeadline.module.css:2]: I would not use absolute `height`

File: components/SubHeadline/SubHeadline.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  padding-right: 25px;
+  padding-left: 25px;
+  align-content: center;
+}
+.subHeader {
+  align-self: end;
+}
+.sort {
+  color: var(--secundary-color);
+  font-size: 15px;
```
Comment (phlgr) [inline components/SubHeadline/SubHeadline.module.css:17]: You should make this responsive wth `em` or `rem`
```suggestion
  font-size: 15rem;
```

File: components/SubHeadline/SubHeadline.module.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  align-self: end;
+  justify-self: end;
+  display: inline;
+  grid-column: 2;
+}
+.date {
+  font-size: 12px;
```
Comment (phlgr) [inline components/SubHeadline/SubHeadline.module.css:24]: Same as the above

**Review by phlgr — CHANGES_REQUESTED** [review]

**Pull request merged**